subsys/net/lib/nrf_cloud/src/nrf_cloud_fota.c

```diff
@@ -536,7 +536,7 @@ int nrf_cloud_fota_unsubscribe(void)
 	struct mqtt_subscription_list sub_list = {
 		.list = sub_topics,
 		.list_count = ARRAY_SIZE(sub_topics),
-		.message_id = NRF_CLOUD_FOTA_SUBSCRIBE_ID
```

Comment:
huh, so it was the same for subscribing and unsubscribing?

Reply:
yeah it wasn't a problem with how we use msg ids... i just added an unsub id to be more complete.